build/NuSpecs/MakeAllAppx.cmd

```diff
@@ -1,6 +1,28 @@
-call ..\..\tools\MakeAppxHelper.cmd x86 release %*
-call ..\..\tools\MakeAppxHelper.cmd x86 debug %*
-call ..\..\tools\MakeAppxHelper.cmd x64 release %*
-call ..\..\tools\MakeAppxHelper.cmd x64 debug %*
-call ..\..\tools\MakeAppxHelper.cmd arm release %*
-call ..\..\tools\MakeAppxHelper.cmd arm debug %*
+@echo off
+REM %~dp0 is position of script file
+
+
+if exist %~dp0"..\..\BuildOutput\Debug\x86\Microsoft.UI.Xaml" (
```

Comment:
@keith Mahoney If for some reason, in the pipelines, if one of these flavors was not populated correctly, we likely would have failed before, but now I think we will silently ignore it ?

Reply:
We never used this script in the Pipeline before. We used to have an equivalent script hardcoded in the yml, which did roughly the same thing.